packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -1,47 +1,120 @@
-// Import React dependencies.
-import React, { useMemo, useState } from 'react'
+// Import dependencies.
+import React, { useMemo, useState, useCallback } from 'react';
 
 // Import the Slate editor factory.
-import { createEditor, BaseEditor } from 'slate'
+import { createEditor, BaseEditor } from 'slate';
 
 // Import the Slate components and React plugin.
-import { Slate, Editable, withReact, ReactEditor } from 'slate-react'
+import {
+  Slate,
+  Editable,
+  withReact,
+  ReactEditor,
+  DefaultElement,
+  RenderLeafProps,
+} from 'slate-react';
+
+// Import Contentful Plugins.
+import { withBoldEvents } from './plugins/Bold';
+import { withItalicEvents } from './plugins/Italic';
+import { withCodeEvents } from './plugins/Code';
+import { withMarksPlugin } from './plugins/Marks';
+import { withUnderlineEvents } from './plugins/Underline';
+import { Leaf } from './plugins/Leaf';
+
+// Import Contentful types.
+import { ContentfulEditor } from './types';
+
+// Import misc.
+import { compose } from './helpers/compose';
 
 type CustomElement = {
   type: 'paragraph';
-  children: CustomText[]
-}
-type CustomText = { text: string }
+  children: CustomText[];
+};
+type CustomText = { text: string };
 
 declare module 'slate' {
   interface CustomTypes {
-    Editor: BaseEditor & ReactEditor
-    Element: CustomElement
-    Text: CustomText
+    Editor: BaseEditor & ReactEditor;
+    Element: CustomElement;
+    Text: CustomText;
```

Comment:
Prettier changes, sorry for this noise.

Reply:
I'm up to that. Usually, Prettier will also read your settings from eslint rules.